tools/coqdep/common.ml

```diff
@@ -10,14 +10,14 @@
 
 open Format
 open Unix
-open Coqdep_lexer
+open Lexer
 (* The whole Minisys module used to be required due to bootstrapping,
    thanks to dune it is not needed anymore and we could just remove it. *)
 open Minisys
```

Comment:
I couldn't work out what this comment intended.

Reply:
Oh that's a long story, thanks for asking. Back in the day, coqdep couldn't depend on any .cma as it had to be built too early in the bootstrap process and was needed in place of ocamldep (so coqdep actually was used to produce the deps for .ml files)

So a few fucntions from the `coq-core.lib` System library were extraced and put into this module, as coqdep was needed to build `coq-core.lib`.

This is not the case anymore, so that whole minisys module can disappear.

Reply:
OK I'll do it in a later patch.